docs/cli-examples.rst

```diff
@@ -0,0 +1,168 @@
+############
```

Comment:
Probably would be good to have an example using `-d/--debug` but that can be a follow-up.